samples/nextjs/scripts/scaffold-component.ts

```diff
@@ -73,9 +73,15 @@ if (manifestOutputPath) {
 function scaffoldComponent() {
   const exportVarName = componentName.replace(/[^\w]+/g, '');
 
-  const componentTemplate = `import { Text } from '@sitecore-jss/sitecore-jss-nextjs';
+  const componentTemplate = `import { Text, Field } from '@sitecore-jss/sitecore-jss-nextjs';
 
-const ${exportVarName} = (props) => (
+interface IProps {
```

Comment:
Subjective, but I like just "Props" here (no "I" prefix)

Reply:
changed to
```
interface ${exportVarName}Props {
  fields: {
    heading: Field<string>;
  };
}
```